charts/k3s/templates/rbac/clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or (not (empty (include "vcluster.serviceMapping.fromHost" . ))) (not (empty (include "vcluster.plugin.clusterRoleExtraRules" . ))) .Values.rbac.clusterRole.create (index .Values.sync "legacy-storageclasses" "enabled") .Values.sync.nodes.enabled .Values.sync.persistentvolumes.enabled .Values.sync.storageclasses.enabled .Values.sync.priorityclasses.enabled .Values.sync.volumesnapshots.enabled -}}
+{{- if or (not (empty (include "vcluster.serviceMapping.fromHost" . ))) (not (empty (include "vcluster.plugin.clusterRoleExtraRules" . ))) .Values.rbac.clusterRole.create (index .Values.sync "legacy-storageclasses" "enabled") .Values.sync.ingresses.enabled .Values.sync.nodes.enabled .Values.sync.persistentvolumes.enabled .Values.sync.storageclasses.enabled .Values.sync.priorityclasses.enabled .Values.sync.volumesnapshots.enabled -}}
```

Comment:
Is this change missing an equivalent in the `clusterrolebinding.yaml` file as well?

...as discovered here: https://github.com/loft-sh/vcluster/issues/755#issuecomment-1255774585

Reply:
i.e. `.Values.sync.ingresses.enabled` hasn't been added on this line:
https://github.com/loft-sh/vcluster/blob/5c3f2fe2e9e0454dff2c52f79d6a0740795fca88/charts/k3s/templates/rbac/clusterrolebinding.yaml#L1

Reply:
@MPV yes, that is correct. Good catch! :+1: 
We would be glad if you create an issue about this, but if you are busy then no problem, we will create one later. We wouldn't mind a PR with a fix either, but again, no pressure :slightly_smiling_face:

Reply:
Oh, I just noticed that the problem already got reported in #765 , so no need to create an issue.

Reply:
@MPV Are you going to create a PR? If not, I can do it.

Reply:
@felipecrs if you plan to create a PR, please leave a comment on #765 so I can assign it to you for tracking and visibility :)